tests/find_and_replace/find_and_replace_store.test.ts

```diff
@@ -808,3 +811,90 @@ test("Selecting a previous match located in a previous sheet will select the las
   expect(getActivePosition(model)).toBe("A1");
   expect(store.selectedMatchIndex).toStrictEqual(0);
 });
+
+describe("replace warnings", () => {
+  beforeEach(() => {
+    setCellContent(model, "A1", "2024");
+    setCellContent(model, "A2", "=DATE(2024, 1, 1)");
+
+    createSheet(model, { sheetId: "sh2", activate: true });
+
+    setCellContent(model, "A1", "2024");
+    setCellContent(model, "A2", "=DATE(2024, 1, 1)");
```

Comment:
it's usually better to keep test setup as minimal as possible and tests as self-contained as possible. Here the second sheet is only useful in 2 tests out of 5 AFAICT.

I'd move the creation of the second sheet only in those 2 tests, even if a few lines are duplicated.

To be fair, I'd even completely remove this `beforeEach` and setup the minimal cells in each test (as an example, the first test only needs one cell to be relevant)